javaagent-api/src/main/javacc/SqlNormalizer.jj

```diff
@@ -22,7 +11,7 @@ options {
 
 PARSER_BEGIN(SqlNormalizer)
 
-package io.opentelemetry.javaagent.instrumentation.jdbc.normalizer;
+package io.opentelemetry.javaagent.instrumentation.api.db.normalizer;
```

Comment:
how about `io.opentelemetry.javaagent.instrumentation.api.normalizer`?

i'm hoping `io.opentelemetry.javaagent.instrumentation.api.db.DbSystem` may go away https://github.com/open-telemetry/opentelemetry-java/issues/1841, which would then leave `.db` empty

also, maybe it should go in `instrumentation-api`? (we can also decide this later)

Reply:
`api.normalizer` is strange. We don't normalize API, do we? I think we need some sort of qualification here.

Reply:
`api.querynormalizer`?

> also, maybe it should go in instrumentation-api? (we can also decide this later)

Hmm, there's no need for this class to be accessible in manual instrumentations. Also, shouldn't we treat `instrumentation-api` kind of like a public API?

Reply:
There is no need, but there can be a "want". Manual instrumentations may want to have access to a standard normalization logic.

Reply:
How about `api.normalize`? or `api.normalization`?

Reply:
Hmm, that still sounds kind of like we're normalizing API.

How about leaving it like it is for now? We can come back to this issue when `DbSystem` gets deleted